.github/workflows/ci.yml

```diff
@@ -77,7 +77,6 @@ jobs:
         # It might be related to boxroot dependency, and we would need to bump
         # up the ocaml-rs dependency
         ocaml_version: ["4.14"]
-        os: ["ubuntu-latest"]
```

Comment:
Note for history: this is the line that seems to require to change the CI settings. LGTM.

Reply:
The target OS is calculated dynamically and is not in use by `strategy.matrix.os` so I removed it yet it messed the naming anyway. Weird.